config/jobs/kubernetes/sig-release/release-branch-jobs/1.12.yaml

```
@@ -448,6 +458,9 @@ periodics:
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: sig-release-1.12-all, sig-release-1.12-blocking, sig-node-kubelet
```

if order of dashboards matters
```suggestion
      testgrid-dashboards: sig-release-1.12-blocking, sig-release-1.12-all, sig-node-kubelet
```